tests/app/forms/validation/test_mutually_exclusive_validator.py

```diff
@@ -25,14 +25,14 @@
     ),
 )
 def test_mutually_exclusive_mandatory_checkbox_raises_ValidationError(
-    answer_permutations, is_mandatory, is_only_checkboxes, error_type
+    answer_permutations, is_mandatory, is_only_checkboxes_or_radios, error_type
```

Comment:
- This test doesn't include an example with a radio answer, the current ones are for checkbox exclusive.
- Should rename the test to remove `checkbox`.
- `ValidationError` in the test name should be `validation_error` (I know it is existing but worth changing while you are here).

Reply:
Renamed the tests and added various radio scenarios (I hope I got it right), thanks.